nextflow.config

```diff
@@ -5,42 +5,78 @@
  * Default config options for all environments.
  */
 
+manifest {
+  name = 'nf-core/sarek'
+  author = 'Maxime Garcia, Szilveszter Juhos'
+  homePage = 'https://github.com/nf-core/sarek'
+  description = 'An open-source analysis pipeline to detect germline or somatic variants from whole genome or targeted sequencing'
+  mainScript = 'main.nf'
+  nextflowVersion = '>=19.10.0'
```

Comment:
Can you verify that the pipeline can most likely still work on earlier versions of nextflow, and therefore you don't need the `!=` option to stop the pipeline if the version doesn't match?

https://www.nextflow.io/docs/latest/config.html#nextflow-version

Reply:
I'll check that out